pegasus/sites.v3/code.org/public/translate/frozen.md

```diff
@@ -0,0 +1,49 @@
+
+1. [Click here](https://amara.org/en/auth/login/?next=/) to create an Amara account.
+1. [Click here to join our Code.org project](https://crowdin.com/project/codeorg) on Crowdin.
+1. From the [main project page](https://crowdin.com/project/codeorg), select your language.
+1. Note your language code in the URL, you'll need this for later. If it's a 4 letter code like "zh-TW" note that the code you'll use later will be in the format "zhTW".
+
+<img src="/images/fit-500/translate-find-code.jpg" style="max-width: 100%;"> 
+
+## Translating the tutorial progression
+The following links all lead directly to the Spanish tutorial. Replace the "es" in the URL with the language code you got earlier. Your translations will be updated on the site at the end of each week. You can watch this [demo video](/files/crowdin.swf) for how to use Crowdin.
+
+1. [tutorial title](https://crowdin.com/translate/codeorg/63/enus-es#68955)
+1. tutorial description [here](https://crowdin.com/translate/codeorg/63/enus-es#68959) and [here](https://crowdin.com/translate/codeorg/40/enus-es#68846)
+1. [tutorial instructions](https://crowdin.com/translate/codeorg/318/enus-es#q=MC+HOC+2016)
+1. [tutorial blocks](https://crowdin.com/translate/codeorg/527/enus-es)
+1. [tutorial hints in callout bubbles](https://crowdin.com/translate/codeorg/41/enus-es#q=callout+craft)
+1. [strings about Frozen on Code.org](https://crowdin.com/translate/codeorg/56/enus-es#q=minecraft) 
+1. [video titles](https://crowdin.com/translate/codeorg/41/enus-es#q=mc_2016)
+1. [video notes](https://crowdin.com/translate/codeorg/46/enus-es#q=mc_2016) - These notes are basically a copy of the video captions. They are shown on the second tab, side by side screenshots of the video. These are intended for slow internet users. Since it is very close to the captions, I suggest completing the translation for the video caption on Amara (see below) and then reusing those translations here.
+
+<img src="/images/fit-500/translate-mc-show-notes.jpg" style="max-width: 100%;">
+
+## Translating the video captions
+
+1. To use Amara, first click on a video link below.
+1. On the left, you'll see a list of languages. If you don't see your language listed, click "Add a new language". A language with a yellow dot means it's been started but incomplete; please go ahead and complete the translation. A language with a green dot means it's been completed and published.
+1. When you begin you'll see the English track on the left and you can enter your translations on the right. Feel free to edit the timing if necessary. When complete, click the Publish button in the top right.
+1. Complete all 5 video translations then email translations@code.org so that we can upload your translations to YouTube. Please include that you completed the Frozen videos and your language.
+1. The links for the videos are below:
+        * [Introduction](https://www.amara.org/en/videos/5fkU0gFODp70/info/frozen-hour-of-code-introduction/)
```

Comment:
Update these 5 videos with 3 videos that are listed under "Frozen (w/ Anna and Elsa)" on code.org/translate/videos